docs/hub/spaces-sdks-docker.md

```diff
@@ -72,10 +65,45 @@ ENV HOME=/home/user \
 # Set the working directory to the user's home directory
 WORKDIR $HOME/app
 
+# Try and run pip command after setting the user with `USER user` to avoid permission issues with Python
+RUN pip install --upgrade pip
+
 # Copy the current directory contents into the container at $HOME/app setting the owner to the user
 COPY --chown=user . $HOME/app
+
+# Download a checkpoint
+RUN mkdir content
+ADD --chown=user https://huggingface.co/<ORG>/<MODEL>/resolve/main/<CHECKPOINT>.ckpt content/<ORG>/<MODEL>/<CHECKPOINT>.ckpt
```

Comment:
Not sure if we should show this, as in most cases we want people to use models through a downstream library and this might confuse users (e.g. `transformers`, `diffusers`, or even `huggingface_hub`)`